src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -1,17 +1,98 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
+﻿using System.Collections.Generic;
 using Microsoft.AspNetCore.SignalR;
+using JabbR_Core.ViewModels;
 
 namespace JabbR_Core.Hubs
 {
+
     public class Chat : Hub
     {
+        public List<string> Rooms;
```

Comment:
Used as a placeholder for `UpdateActivity(UserData user, List<string> rooms)`. Works without the parameter but wanted to keep signature the same.